README.md

```diff
@@ -50,8 +50,9 @@ client = ReportingApiClient(server_url=SERVER_URL, key=API_KEY)
 ```
 
 Besides the microgrid_id, component_ids, and metrics, start, and end time,
-you can also set the sampling period for resampling using the `resolution` parameter.
-For example, to resample data every 15 minutes, use a `resolution` of 900 seconds. The default is 1 second.
+you can also set the sampling period for resampling using the `resampling_period`
```

Comment:
Just noting that the SDK uses `sampling_period`, but I think the term you chose is more accurate, so I would keep it.